From: Alberto Bertogli <albertito@blitiri.com.ar>
Date: Thu, 23 Jul 2009 15:03:37 +0000 (-0300)
Subject: libfiu: Put the version in the Makefile and use it to generate libfiu.pc
X-Git-Url: http://blitiri.com.ar/git/?p=libfiu;a=commitdiff_plain;h=d5badedfb9c573be39fb85c60db1a9512d64f338

libfiu: Put the version in the Makefile and use it to generate libfiu.pc

In future commits we will make further use of $(LIB_VER).

Signed-off-by: Alberto Bertogli <albertito@blitiri.com.ar>
---

diff --git a/libfiu/Makefile b/libfiu/Makefile
index e312976..3392992 100644
--- a/libfiu/Makefile
+++ b/libfiu/Makefile
@@ -27,6 +27,8 @@ else
 	NICE_CC = $(CC)
 endif
 
+LIB_VER=0.12
+
 
 default: all
 
@@ -35,7 +37,8 @@ all: libs libfiu.pc
 libfiu.pc: build-flags libfiu.pc.in
 	@echo "generating libfiu.pc"
 	@cat libfiu.pc.in | \
-		sed 's@++PREFIX++@$(PREFIX)@g' \
+		sed 's@++PREFIX++@$(PREFIX)@g' | \
+		sed 's@++LIB_VER++@$(LIB_VER)@g' \
 		> libfiu.pc
 
 libs: libfiu.so libfiu.a
diff --git a/libfiu/libfiu.pc.in b/libfiu/libfiu.pc.in
index 4f1907d..de3e08b 100644
--- a/libfiu/libfiu.pc.in
+++ b/libfiu/libfiu.pc.in
@@ -6,7 +6,7 @@ includedir=${prefix}/include
 Name: libfiu
 Description: Fault injection in userspace
 URL: http://blitiri.com.ar/p/libfiu/
-Version: 0.12
+Version: ++LIB_VER++
 Libs: -L${libdir} -lfiu
 Cflags: -I${includedir}
 
